### DISCUSSION TITLE
Favorite (Plex List) inside single page (movie or series)

### DISCUSSION THREAD

**Discussion by meteoro** [seerr-team/seerr#843] [Ideas]
Sometimes when searching for a film or series that hasn't been released yet and I want to save, I miss a favorite button like on the listing page (showcase).

<img width="198" alt="Screenshot 2024-06-27 at 00 13 17" src="https://github.com/Fallenbagel/jellyseerr/assets/1403424/8bb71e62-b885-47de-ac6a-3786a2dc329f">

<img width="1170" alt="Screenshot 2024-06-27 at 00 17 48" src="https://github.com/Fallenbagel/jellyseerr/assets/1403424/7dab8357-5a0a-4506-b7f7-a97fa99400a9">

**Comment by fallenbagel** [comment]
https://github.com/Fallenbagel/jellyseerr/pull/781
